cmd/root.go

```diff
@@ -28,6 +28,7 @@ var (
 	flagConfiggin      string
 	flagLightOpinions  string
 	flagDarkOpinions   string
+	flagJSON           bool
```

Comment:
Done with commit 46881bfff4c6d0b74fced5e03fa0dd0456d43580.
Example of yaml output attached to the ticket.